pom.xml

```diff
@@ -81,7 +81,7 @@
 				<dependency>
 					<groupId>com.bitwig</groupId>
 					<artifactId>extension-api</artifactId>
-					<version>[13,)</version>
+					<version>[10,)</version>
```

Comment:
On the branch api-14 is should be version `14` as the requirement I'm not sure if we really benefit using `[14,)`